purchase_order_import/tests/test_order_response_import.py

```diff
@@ -413,7 +413,8 @@ def test_13(self):
         ]
         self.OrderResponseImport.process_data(data)
         self.assertEqual(self.purchase_order.state, "purchase")
-        self.assertEqual(len(self.purchase_order.picking_ids), 2)
+        self.assertEqual(len(self.purchase_order.picking_ids), 1)
+        self.assertTrue(self.purchase_order.picking_ids.backorder_id)
```

Comment:
Wow isn't the picking backorder marked as a picking of the purchase order anymore? Do you know if this functional change is intentional in Odoo 14?